shared_infra/lambdas/common/src/wellcome_lambda_utils/miro_utils.py

```diff
@@ -0,0 +1,20 @@
+import re
+
+
+class MiroImage:
+    def __init__(self, image_info):
+        self.collection = image_info['collection'].split("-")[-1]
+        self.image_data = image_info['image_data']
+
```

Comment:
It would be helpful to add a `__repr__` method. This gets called to print the debug representation in tracebacks, interactive console, etc. Right now you just get `<__main__.MiroImage object at 0x107d678d0>`.

Something like:

```python
def __repr__(self):
    image_info = {'collection': self.collection, 'image_data': self.image_data}
    return f'{type(self).__name}(image_info={image_info!r})'
```

Ideally it’s a string you could `eval()` to get an equivalent object.